vllm/envs.py

```diff
@@ -11,6 +11,7 @@
     VLLM_USE_TRITON_FLASH_ATTN: bool = False
     LOCAL_RANK: int = 0
     CUDA_VISIBLE_DEVICES: Optional[str] = None
+    HIP_VISIBLE_DEVICES: Optional[str] = None
```

Comment:
What about only exposing `CUDA_VISIBLE_DEVICES` to the vLLM users and we internally convert it to `HIP_VISIBLE_DEVICES` when using the AMD GPUs?

Reply:
I've done that for these tests for now, we can revisit how we use `CUDA_VISIBLE_DEVICES` generally in a follow-up PR. Copying over any changes to `CUDA_VISIBLE_DEVICES` to `HIP_VISIBLE_DEVICES` is not portable in general. Probably an abstraction layer that either routes device changes to `HIP_VISIBLE_DEVICES` or `CUDA_VISIBLE_DEVICES` is best.

Reply:
In general we cannot expect ROCm to recognize `CUDA_VISIBLE_DEVICES`. Unfortunately we currently don't have a unified interface for setting `CUDA_VISIBLE_DEVICES`: each use case (Ray executor, MP executor, custom all reduce, tests, etc) does it in its own way, so there will need to be quite a bit of refactoring, and even then, I'm not sure if there's a better solution than just always making sure that changes to `CUDA_VISIBLE_DEVICES` <=> changes to `HIP_VISIBLE_DEVICES`.